app/editor/src/components/tabs/TabMenu.tsx

```diff
@@ -0,0 +1,20 @@
+import { Row } from 'components/row';
+import React from 'react';
+
+import * as styled from './styled';
+
+/**
+ * The element that groups the various navigation bar items together.
+ * @param children the navigation bar items
+ * @returns navigation bar group
+ */
+export const TabMenu: React.FC<React.HTMLAttributes<HTMLDivElement>> = ({
+  children,
+  className,
+}) => {
+  return (
+    <styled.TabMenu className={`${className ?? 'tab-menu'}`}>
```

Comment:
I'm doing this `className` thing in most places now.  It gives us default values for components, which will make it easier to override styling in some places.